imagekit/cachefiles/backends.py

```diff
@@ -74,7 +75,7 @@ def set_state(self, file, state):
         if state == CacheFileState.DOES_NOT_EXIST:
             self.cache.set(key, state, self.existence_check_timeout)
         else:
-            self.cache.set(key, state)
+            self.cache.set(key, state, settings.IMAGEKIT_CACHE_TIMEOUT)
```

Comment:
Why introduce a timeout here?  I think it's actually better to not let this expire...